js/viz/range_selector.d.ts

```diff
@@ -54,8 +58,6 @@ export {
 /** @public */
 export type BackgroundImageLocation = 'center' | 'centerBottom' | 'centerTop' | 'full' | 'leftBottom' | 'leftCenter' | 'leftTop' | 'rightBottom' | 'rightCenter' | 'rightTop';
 /** @public */
-export type ValueChangedCallMode = 'onMoving' | 'onMovingComplete';
-/** @public */
```

Comment:
Generally we can not just remove `export` statements because this cause breaking changes.

Add reexport (place it at the beginning of `export`'s block)

```ts
export {
  ValueChangedCallMode,
} from '../common';
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/22785/commits/535c3bde2ed65328bd69ae6a8b75bf17e0f749b2